Zend/zend_vm_execute.h

```diff
@@ -15802,21 +15812,29 @@ static ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL ZEND_NEW_SPEC_VAR_HANDLER(ZEND_OPCO
 
 	constructor = Z_OBJ_HT_P(result)->get_constructor(Z_OBJ_P(result));
 	if (constructor == NULL) {
-		ZEND_VM_JMP(OP_JMP_ADDR(opline, opline->op2));
+		if (EXPECTED(opline->extended_value == 0)) {
```

Comment:
This won't work with ZEND_EXT_NOP (or ZEND_TICK) inserted between NEW and DO_FCALL,